modules/github.js

```diff
@@ -12,13 +19,86 @@ const echo = require("./echo");
 // -----------------------------------------------------------------------------------------
 
 const github = {
+    // -----------------------------------------------------------------------------------------
+    // #region Properties
+    // -----------------------------------------------------------------------------------------
+
+    _prompt: null,
+    _token: null,
     andcultureOrg: "AndcultureCode",
     apiRepositoriesRouteParam: "repos",
     apiRootUrl: "https://api.github.com",
+    configAuthConfigPath: upath.toUnix(path.join(os.homedir(), ".netrc")), // Path to octokit-auth-netrc configuration
+    configAuthDocsUrl:
+        "https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token",
+    configAuthTokenUrl: "https://github.com/settings/tokens/new",
+
+    // #endregion Properties
+
+    // -----------------------------------------------------------------------------------------
+    // #region Public Methods
+    // -----------------------------------------------------------------------------------------
+
+    /**
+     * Sets login token for github api
+     * @param {string} token github personal access token
+     */
+    configureToken(token) {
+        let contents = "\r\nmachine api.github.com\r\n";
```

Comment:
nitpick: consolidate this string assignment
```js
const contents = `
machine api.github.com
    login ${token}

`;
```

Reply:
I actually went back and forth on that. Being you mentioned it, I'll switch over. Thanks!

Reply:
Ah, you know what. I thought of a better way to work around the unfortunate outdenting while also leveraging your recommendation. Check it out

Reply:
Yeah, I'd prefer the array approach to maintain a normal indentation level ☝️